src/families/tron/types.js

```diff
@@ -196,6 +205,7 @@ export type TronResourcesRaw = {|
     bandwidth: ?DelegatedFrozenInfoRaw,
     energy: ?DelegatedFrozenInfoRaw
   },
+  delegatedResources: DelegatedResources[],
```

Comment:
`DelegatedResources` is not serializable (meaning `JSON.parse(JSON.stringify(delegatedResources))` is not equivalent to `delegatedResources`) so we need to have a `DelegatedResourcesRaw` that is covered in the "{from/to}TronResourceRaw" functions.

Reply:
Ok! I will do. Thanks!